Makefile

```diff
@@ -2,10 +2,10 @@
 .DEFAULT_GOAL := help
 
 test: # Run Unit Test
-	python -m unittest tests
+	pytest tests
```

Comment:
we do not use any other dependencies for tests currently and the CI workflow also does not install any dependencies and just runs a single file: https://github.com/deta/deta-python/blob/master/.github/workflows/pull_request.yml#L24-L31

we need to decide internally if we should use `pytest`, if yes, we also need to update the other tests to use `pytest` so that it's consistent and update the PR workflow.

Reply:
ok

pytests is recommended way to test aiohttp. And it makes the developer experience better.

But if you decided not to use it, there is a way for testing aiohttp with unittest and I can rewrite tests:
https://docs.aiohttp.org/en/stable/testing.html#unittest

Reply:
`pytests` should be fine, we'll use it from now on for tests. You don't have to update the github workflow or the other tests. I will do that in a different PR.